package.json

```diff
@@ -54,6 +54,16 @@
   "eslintConfig": {
     "extends": "react-app"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.+(js|jsx|json|css|md)": [
```

Comment:
let's revert this and tackle it in a standalone PR.

also ts and tsx is missing, and won't this pattern match repeated patterns like `something.jsjs`?

Reply:
Yeah, you're right if the library it's using regex. Good catch!

![image](https://user-images.githubusercontent.com/11761240/88197094-fe39d480-cc0f-11ea-8084-fa62645c49fc.png)
![image](https://user-images.githubusercontent.com/11761240/88197133-0abe2d00-cc10-11ea-869e-a103ab627937.png)
https://regex101.com/

Reply:
oh ok, sorry about that because I followed a tutorial. I've reverted the changes, so we can address this in a new PR

Reply:
I created an Issue about Husky https://github.com/osrf/romi-dashboard/issues/68